renderlib/gesture/gesture.h

```diff
@@ -0,0 +1,388 @@
+#pragma once
+#include "glad/glad.h"
+
+#include <glm/glm.hpp>
+#include <glm/gtx/string_cast.hpp>
+
+#include "BoundingBox.h"
+#include "CCamera.h"
+#include "SceneView.h"
+#include "graphics/gl/Util.h"
+#include "graphics/glsl/GLGuiShader.h"
+
+#include <memory>
+#include <vector>
+
+class GLGuiShader;
+class Scene;
+class RenderSettings;
+
+// https://maxliani.wordpress.com/2021/06/06/offline-to-realtime-gesture/
+
+struct Gesture
+{
+  // all mouse positions are in window coordinates
+  // premultiplied by devicepixelratio in the gui layer
+  // with 0,0 at TOP left.
+  struct Input
+  {
+    // User Input
+    static constexpr size_t kButtonsCount = 3;
+
+    // During app initialization query the OS accessibility settings how the user configured the
+    // double-click duration. Developer: never hardcode this time to something that feels
+    // right to you.  The initial value is just a nonzero default that is meant to be overriddent.
+    double doubleClickTime = 0.5; //< Mouse double-click time in seconds.
+    void setDoubleClickTime(double seconds) { doubleClickTime = seconds; }
```

Comment:
The blog posts mentions querying the OS APIs to set this, did you ever end up implementing that?

Reply:
Yep, you can see that code in GLView3D.cpp